packages/flutter_tools/lib/src/windows/visual_studio.dart

```diff
@@ -294,22 +299,49 @@ class VisualStudio {
         ...requirementArguments,
       ], encoding: encoding);
       if (whereResult.exitCode == 0) {
-        final List<Map<String, dynamic>> installations =
-            (json.decode(whereResult.stdout) as List<dynamic>).cast<Map<String, dynamic>>();
-        if (installations.isNotEmpty) {
+        final List<Map<String, dynamic>>? installations = _tryDecodeVswhereJson(whereResult.stdout);
+        if (installations != null && installations.isNotEmpty) {
           return VswhereDetails.fromJson(validateRequirements, installations[0]);
         }
       }
     } on ArgumentError {
       // Thrown if vswhere doesn't exist; ignore and return null below.
     } on ProcessException {
       // Ignored, return null below.
-    } on FormatException {
-      // may be thrown if invalid JSON is returned.
     }
     return null;
   }
 
+  List<Map<String, dynamic>>? _tryDecodeVswhereJson(String vswhereJson) {
+    late List<dynamic> result;
+    late FormatException error;
+    try {
+      // vswhere.exe is known to encode its output incorrectly, resulting in
```

Comment:
Nit: "Some versions of vswhere.exe are know to [....]"

Reply:
Fixed!